cmd/Dockerfile

```diff
@@ -76,19 +77,21 @@ COPY --from=build-debug /app/app /bin/app
 
 ENTRYPOINT ["dlv", "--listen=:2345", "--headless=true", "--api-version=2", "--accept-multiclient", "exec", "/bin/app"]
 
-FROM build-env as build-production-nocgo
+FROM build-env as build-transit-production-nocgo
```

Comment:
Why the name change?
This `From` and the next are specifically for building the `transit` script and now it passes the extra loader arguments. Hence, I renamed it to reflect that it is only for the `transit` script.